taskcluster/mozillavpn_taskgraph/transforms/beetmover.py

```diff
@@ -0,0 +1,99 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+import os.path
+
+from taskgraph.transforms.base import TransformSequence
+from taskgraph.util.schema import resolve_keyed_by
+
+transforms = TransformSequence()
+
+
+@transforms.add
+def add_beetmover_worker_config(config, tasks):
+    for task in tasks:
+        app_name = "vpn"
+        worker_type = task["worker-type"]
+        task_name = task["name"]
+        task_label = f"{worker_type}-{task_name}"
+        l3_relpro = (
+            config.params["level"] == "3"
+            and config.params["tasks_for"] in task["run-on-tasks-for"]
+        )
+        bucket = "release" if l3_relpro else "dep"
+        build_id = config.params["moz_build_date"]
+        build_type = task["attributes"]["build-type"]
+        build_os = os.path.dirname(build_type)
+        app_version = config.params["version"]
+        candidates_path = os.path.join(
+            "pub", app_name, "candidates", app_version, build_id, build_os
```

Comment:
So I guess we're going to implement the task to copy these over to the `releases` dir as a follow-up? We'll want to refactor this to re-use logic then, but happy to leave that for later.

Reply:
Makes sense. I'll do it later